draft-ietf-quic-transport.md

```diff
@@ -1408,8 +1408,6 @@ This system allows a server to process packets with unsupported versions without
 retaining state.  Though either the Initial packet or the Version Negotiation
 packet that is sent in response could be lost, the client will send new packets
 until it successfully receives a response or it abandons the connection attempt.
-As a result, the client discards all state for the connection and does not send
-any more packets on the connection.
```

Comment:
Why remove this?  It doesn't seem to be responsive to the concerns that the rest of the change addresses.

Reply:
It's unrelated and could easily be elided.
I read through the whole top-level section while working on this PR, and I thought this looked out of place (it's the "Remove a sentence (apparently) about handling version negotiation packets from the section that's supposed to be about sending them.").  I could be mistaken about that, though, and we can split it out if that's preferred.

Reply:
Please try to focus on targeted fixes.  That makes this process a lot easier to manage.

Reply:
Before I commit another faux pas, is it preferred to force-push or push a revert commit (or something else)?

Reply:
I don't personally mind.  In this repo, we have let the commit history reflect the history of the issues.  So push as you like.